src/content/docs/tutorial-customer-experience/bottom-funnel-analysis.mdx

```diff
@@ -0,0 +1,190 @@
+  - /docs/new-relic-solutions/observability-maturity/customer-experience/bofta-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/customer-experience/bottom-funnel-analysis-customer-journey-guide
+---
+
+import cxBotfaFunnel from 'images/cx_diagram_botfa-funnel.webp'
+
+import cxBotfaEcommerce from 'images/cx_diagram_botfa-ecommerce.webp'
+
+import cxBotfaCarInsurance from 'images/cx_diagram_botfa-car-insurance.webp'
+
+import cxBotfaPagesActions from 'images/cx_diagram_pages-actions.webp'
+
+import cxBotfaDashboard from 'images/cx_diagram_botfa-dashboard.webp'
+
+import cxBotfaKpiTracking from 'images/cx_diagram_botfa-kpi-tracking.webp'
+
+Even though [improving customer experience](/docs/tutorial-customer-experience/improve-customer-experience/) is often done comprehensively, you may find that focusing your efforts on improving conversion can also provide a large increase in customer experience quality. This is what's known as bottom-of-the-funnel analysis is about improving conversion by focusing on performance at the end of the user journey instead of the customer experience as a whole. 
+
+<img
+  title="User journey funnel"
+  alt="User journey funnel"
+  src={cxBotfaFunnel}
+/>
+
+Most sites and apps are built with one or more purposes in mind, and where there's a purpose, there's also likely a user journey. For example:
+
+| Purpose                                           | User journey                                         |
+| ------------------------------------------------- | ---------------------------------------------------- |
+| Brand awareness                                   | Download a whitepaper                                |
+| Customer support                                  | Raise a support ticket                               |
+| Customer services (registration, forms, payments) | Make a payment                                       |
+| Entertainment                                     | Stream a movie                                       |
+| Goods                                             | Buy clothes and accessories                          |
+| Informational purposes                            | Find a support program in your state                 |
+| Lead generation                                   | Join distribution list to get promotions and coupons |
+| Outreach                                          | Get information about events in your town            |
+| Services (travel, rentals, bookings)              | Book a flight                                        |
+| Social media                                      | Share a selfie                                       |
+
+When a user completes a journey, it's reffered to as a conversion. All conversions have a value, and the best way to improve the conversion rate is to start at the bottom of the funnel, when the intent to complete the user journey is clear.
```

Comment:
"referred"

Reply:
I thought a conversion was more specific to someone creating an account or starting to pay for something?

Reply:
Thanks, that's one of those words that usually gets me. I think in this case, conversion means what's written here, but I'll double check with OMA folks.